published_bucket.tf

```
@@ -303,8 +256,7 @@ data "aws_iam_policy_document" "pdm_read_pii_and_non_pii" {
     ]
 
     resources = [
-      data.terraform_remote_state.common.outputs.published_bucket.arn,
-      data.terraform_remote_state.common.outputs.published_nonsensitive.arn,
+      data.terraform_remote_state.common.outputs.published_bucket.arn
```

you could make this diff cleaner by keeping the trailing commas, it will keep future changes cleaner to
e.g.
```suggestion
      data.terraform_remote_state.common.outputs.published_bucket.arn,
```